### ISSUE TITLE
bibRef author not complete

### ISSUE THREAD

**Issue by myrmoteras** [gsautter/goldengate-imagine#319]
this seems to be a common omission in the detection of the authority name
[zootaxa.4341.2.3.pdf](https://github.com/gsautter/goldengate-imagine/files/1429890/zootaxa.4341.2.3.pdf)

![image](https://user-images.githubusercontent.com/4609956/32212577-c5789536-be17-11e7-805b-e5f772a3dc57.png)

**Comment by gsautter** [comment]
Yes, and a weird one ... initials "V.A." of the second author appear to have tagged just like supposed to. I'll try and prevent such cutoffs in the new patterns designed in the course of the RefParse makeover.
